instant/flags.vim

```diff
@@ -33,8 +33,8 @@ endif
 
 
 " Shout if maktaba is too old. Done here to ensure it's always triggered.
-if !maktaba#IsAtLeastVersion('1.10.0')
-  call maktaba#error#Shout('Codefmt requires maktaba version 1.10.0.')
+if !maktaba#IsAtLeastVersion('1.12.0')
```

Comment:
Could you also comment what feature needs 1.12?

Reply:
That's the second bullet in the commit message. Maktaba prior to 1.12.0 doesn't work on any relatively modern vim or neovim because of the linked issue. The only reason 1.10.0 works in travis is that [vim on xenial is 7.4.1689](https://packages.ubuntu.com/search?keywords=vim-gnome) which is prior to the breakage.

I guess the other way to approach this is to say that vim-codefmt only requires 1.10.0 in `instant/flags.vim` but the neovim tests require 1.12.0 in `.travis.yml`. Would that be preferable to you?

Reply:
Yep, the reason is fine, I just meant to have it as a comment in the file itself w/ some kind of self-contained link to the reason.

Reply:
Oh, gotcha